src/java.base/share/classes/jdk/internal/reflect/Reflection.java

```diff
@@ -56,7 +56,7 @@ public class Reflection {
         fieldFilterMap = Map.of(
             Reflection.class, ALL_MEMBERS,
             AccessibleObject.class, ALL_MEMBERS,
-            Class.class, Set.of("classLoader", "classData", "modifiers"),
+            Class.class, Set.of("classLoader", "classData", "modifiers", "isPrimitive"),
```

Comment:
I think the field is named `isPrimitive`, right?

Reply:
The method is isPrimitive so I think I had to give the field isPrimitiveType as a name, so this is wrong.

Reply:
I don't know if we have a style guide that covers this, but I believe the method and field could both be named `isPrimitive`.

Reply:
I would personally name such a boolean field `primitive`, but I don't have a strong preference on the field naming as long as its references in tests and other locations are correct. In addition, I believe this field may soon be widened to carry more hotspot-specific flags (such as hidden, etc.) so the name is bound to change.

Reply:
I like 'primitive'.  'hidden' is also a possibility to add to this and give it the same treatment.  I didn't do that one here to limit the changes and I haven't seen all the calls to isHidden so would need to find out how to measure the effects of that change.